llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -6824,7 +6824,7 @@ SDValue SelectionDAG::FoldConstantArithmetic(unsigned Opcode, const SDLoc &DL,
             continue;
           }
           APInt Val = cast<ConstantSDNode>(Op)->getAPIntValue();
```

Comment:
Can we change the preceding line to `const APInt &Val` instead? `getAPIntVal` returns a reference

Reply:
Done. Thanks.